packages/bigquery/src/index.js

```diff
@@ -450,6 +724,39 @@ BigQuery.prototype.job = function(id) {
  * });
  *
  * //-
+ * // Positional and named SQL parameters are supported.
```

Comment:
Can we change this a little to add a comment above each example?

e.g.

Positional SQL parameters are supported

```js
// positional example
```

Or if you prefer to name them, that's also supported.

```js
// named example
```